core/google/cloud/iam.py

```diff
@@ -209,9 +196,9 @@ def to_api_repr(self):
         if self.version is not None:
             resource['version'] = self.version
 
-        bindings = self._role_bindings()
-
-        if bindings:
-            resource['bindings'] = bindings
+        if len(self.bindings) > 0:
+            resource['bindings'] = [
+                {'role': role, 'members': members}
```

Comment:
Might as well check for empty members here since that is also an API error. It's not likely but users accidentally end up with empty members via use of `remove`.